app/views/unattended/poap/disklayout.erb

```diff
@@ -0,0 +1,5 @@
+<%#
+kind: ptable
+name: NX-OS default fake
+%>
+# Not required for NX-OS POAP provisioning.
```

Comment:
@fernandocarolo That sounds fair enough, basically the same approach as ZTP. Please submit a PR to community-templates with that then :smile: